pages/mesosphere/dcos/2.1/tutorials/windows/beta-features/index.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: layout.pug
+navigationTitle: Windows Usage examples 
+title: Windows Usage examples 
+menuWeight: 40
+excerpt: Windows Usage examples
+model: /mesosphere/dcos/2.1/data.yml
+render: mustache
+---
+
+# Windows Usage examples
+
+This short tutorial will help you to install and configure a mixed DC/OS cluster that includes a Windows node.
+
+## Demo Running Windows Docker Containers
+
+Install [Marathon-LB](/mesosphere/dcos/services/marathon-lb/1.14/mlb-install/) from the DC/OS service catalogue and you will have the ability to access the app using http://<linuxpublicagentipaddress>
+
+Use the Marathon template to configure and start a [Windows workload app](https://github.com/dcos/windows-test-workload/blob/master/windows-docker-container/windows-docker-app.json).
+
+There will be a link to Demo video.
```

Comment:
@cmastr  Are we adding videos to the documentation?
If yes, this would be this video: https://drive.google.com/drive/folders/1RWKWO3D7lC5jufzlL2DtRh3CQSJcTBfW

Reply:
ah let me check!